packages/hash/api/src/graphql/resolvers/knowledge/page/page.ts

```diff
@@ -72,7 +72,9 @@ export const pages: ResolverFn<
   QueryPagesArgs
 > = async (_, { ownedById }, { dataSources: { graphApi }, userModel }) => {
   const accountEntityId = ownedById
-    ? entityIdFromOwnedByIdAndEntityUuid(systemUserAccountId, ownedById)
+    ? // When type branding, we would need to cast `ownedById` to an `EntityUuid`.
+      // We may want to special-case this casting and have an explicit function to do it.
+      entityIdFromOwnedByIdAndEntityUuid(systemUserAccountId, ownedById)
```

Comment:
Are we not type branding? What's going on here?

Reply:
2e79e039d40883c9728dd67f49e50266b1d43ebf sorry, I missed my own 'todo' here :)